### ISSUE TITLE
Regression Test MP Unit Information

### ISSUE THREAD

**Issue by vishnunavuluri** [US-EPA-CAMD/PI-1#4928]
The scope of this ticket shall be regression testing the following functionality for MP Unit Information (Both Global-view and Workspace):

- Import
- Export 
- Evaluate
- Create
- View
- Edit

Note: Includes Unit Fuels, Unit Controls and Unit Capacities Data

**Comment by billmiller13** [comment]
![image.png](https://images.zenhubusercontent.com/62f3bc25cca3fb78227f8f0e/960da0e8-e3cd-4e85-9444-863985a37c78)

**Comment by billmiller13** [comment]
In workspace view I successfully tested Create, Edit, and View for a Unit Fuels, Unit Capacities, and a Unit Controls record.  I exported the MP and saw these records that I created and edited.  Then I ran an import with a new one of each these types of records and confirmed they appeared in the UI Data Table.

In global view I viewed each of these types of records in the MP Unit Information section, exported the MP, then confirmed each of the records I saw in the UI existed in the export JSON.
